packages/web/docs/src/pages/product-updates/index.mdx

```diff
@@ -1,11 +0,0 @@
-import { useData } from '@theguild/components'
-import { ProductUpdates } from '../../components/product-updates'
-
-export { getStaticProps } from '../../components/product-updates'
-
-export const ProductUpdatesPage = () => {
-  const { changelogs } = useData()
-  return <ProductUpdates changelogs={changelogs} />
-}
-
-<ProductUpdatesPage />
```

Comment:
I put it in `pages/product-updates.mdx`

Reply:
with this change we no longer have duplicates Product Update -> Product Update 
<img width="489" alt="image" src="https://github.com/user-attachments/assets/452ad649-63e4-4d3b-81d5-6a37f269a9ce">

Reply:
Can we please completely remove the sub-items of the product updates in this menu and always link to the overview page?

Reply:
fixed 
<img width="491" alt="image" src="https://github.com/user-attachments/assets/131559d5-601f-4855-8191-b8ae8e5b5b81">